src/wireviz/Harness.py

```diff
@@ -321,8 +321,10 @@ def output(self, filename: (str, Path), view: bool = False, cleanup: bool = True
             for row in listy[1:]:
                 file.write('<tr>')
                 for i, item in enumerate(row):
+                    item_str = item.replace('\u00b2', '<sup>2</sup>')
```

Comment:
I wonder if using [`&sup2;`](https://www.toptal.com/designers/htmlarrows/punctuation/superscript-two/) will work better in this case, since it should preserve the correct character when copying & pasting from the HTML file?

Reply:
Your call.  I'm not an HTML guy, I just knew how to fix it.

Reply:
Yup, [just tried it](https://www.w3schools.com/code/tryit.asp?filename=GGTWHOIGBPI6); please use `&sup2;` :)
Currently there's a conflict, but for such a small change, it should be easy to resolve.
Thanks!